### PR TITLE
Bunch of 21w38a mappings.

### PR DESCRIPTION
**Pull request by LambdAurora**
- Move structure piece stuff to `net.minecraft.structure.piece` instead of `net.minecraft.structure`.

### CODE REVIEW

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/server/command/dev/ChaseCommand.mapping [hunk 1]
```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/unmapped/C_rdsjnijy net/minecraft/server/command/dev/ChaseCommand
+	METHOD m_kdvwlhrz executeStop (Lnet/minecraft/unmapped/C_krlbnnep;)I
+		ARG 0 source
+	METHOD m_kxbqrpdf executeStart (Lnet/minecraft/unmapped/C_krlbnnep;)Z
+		ARG 0 source
+	METHOD m_ldthwyxo (Lnet/minecraft/unmapped/C_krlbnnep;Ljava/lang/String;I)I
+		ARG 0 source
+		ARG 2 port
+	METHOD m_miuttkrz (Lcom/mojang/brigadier/CommandDispatcher;)V
```
Comment (IotaBread) [inline mappings/net/minecraft/server/command/dev/ChaseCommand.mapping]:
```suggestion
	METHOD m_miuttkrz register (Lcom/mojang/brigadier/CommandDispatcher;)V
```

**Review by Bubblie01 — APPROVED** [review]
Everything looks good

File: mappings/net/minecraft/structure/piece/StructurePieceSerializationContex.mapping [hunk 2]
```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/unmapped/C_ozrxzywy net/minecraft/structure/piece/StructurePieceSerializationContex
```
Comment (IotaBread) [inline mappings/net/minecraft/structure/piece/StructurePieceSerializationContex.mapping]:
```suggestion
CLASS net/minecraft/unmapped/C_ozrxzywy net/minecraft/structure/piece/StructurePieceSerializationContext
```

File: mappings/net/minecraft/structure/piece/StructurePieceType.mapping [hunk 3]
```diff
@@ -56,5 +56,18 @@ CLASS net/minecraft/unmapped/C_amkyvqte net/minecraft/structure/StructurePieceTy
...
 	METHOD m_lawegmeo load (Lnet/minecraft/unmapped/C_ozrxzywy;Lnet/minecraft/unmapped/C_hincwlhh;)Lnet/minecraft/unmapped/C_yiqopooc;
+		ARG 1 context
 		ARG 2 nbt
+	CLASS C_cyrdazlb
+		METHOD m_lerqcthn (Lnet/minecraft/unmapped/C_wviludhr;Lnet/minecraft/unmapped/C_hincwlhh;)Lnet/minecraft/unmapped/C_yiqopooc;
+			ARG 1 structureManager
+			ARG 2 nbt
+	CLASS C_urjppodc
+		METHOD m_oqgqhxyg (Lnet/minecraft/unmapped/C_hincwlhh;)Lnet/minecraft/unmapped/C_yiqopooc;
+			ARG 1 nt
```
Comment (IotaBread) [inline mappings/net/minecraft/structure/piece/StructurePieceType.mapping]:
```suggestion
			ARG 1 nbt
```

**Review by AlsoSylv — APPROVED** [review]

File: mappings/net/minecraft/util/profiler/jfr/JfrRecording.mapping [hunk 4]
```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_ythdruyh
+CLASS net/minecraft/unmapped/C_ythdruyh net/minecraft/util/profiler/jfr/JfrRecording
```
Comment (IotaBread) [inline mappings/net/minecraft/util/profiler/jfr/JfrRecording.mapping]: This is `JfrStatsParser` in Mojmap, I don't think JfrRecording is a right name

**Review by IotaBread — CHANGES_REQUESTED** [review]

**Review by NoComment1105 — APPROVED** [review]

**Review by IotaBread — APPROVED** [review]

**Review by AlsoSylv — APPROVED** [review]

**Pull request merged**